resources/assets/uad_lists.json

```
@@ -3818,11 +3827,11 @@
   {
     "id": "com.motorola.coresettingsext",
     "list": "Oem",
-    "description": "Core Settings extension\nSafe to remove (no bootloop) but its usefulness remains unkown.\n",
+    "description": "Core Settings extension\nSafe to remove (no bootloop) but its usefulness remains unkown.\nIt's an app for random settings, but it's unused?",
     "dependencies": [],
     "neededBy": [],
     "labels": [],
-    "removal": "Recommended"
+    "removal": "Advanced"
```

```suggestion
    "removal": "Expert"
```
If the implications are unknown and it's related to settings, maybe it should be expert instead. "Random settings" is too vague.

---

this app is safe to remove so better keep it in advanced?

---

if someone who first wrote this description, really remove this app

---

But settings can be a vital part of the system, which is what `Expert` is for.